media-libs/libheif/libheif-1.13.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools xdg multilib-minimal
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/strukturag/libheif.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/strukturag/libheif/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
+fi
+
+DESCRIPTION="ISO/IEC 23008-12:2017 HEIF file format decoder and encoder"
+HOMEPAGE="https://github.com/strukturag/libheif"
+
+LICENSE="GPL-3"
+SLOT="0/1.12"
+IUSE="+aom gdk-pixbuf go rav1e test +threads x265"
+REQUIRED_USE="test? ( go )"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	test? (
+		<dev-cpp/catch-3
```

Comment:
Could you link a bug in here so we have a reference

Reply:
Done. Also reference is added in src_prepare() where catch.hpp is replaced with system installed.

Reply:
The unbundling of catch was in 
https://gitweb.gentoo.org/repo/gentoo.git/commit/media-libs/libheif?id=477b7bfe0e87aa72022fc686fe8697d9923638ad
for libheif-1.12.0-r3

Reply:
Oh, right, thanks